products/coldfusion.md

```diff
@@ -18,32 +18,49 @@ releases:
     releaseDate: 2023-05-17
     eol: 2028-05-16
     extendedSupport: 2029-05-16
-    link: null # does not exist yet, will probably become valid after the first update
+    latestReleaseDate: 2023-10-05
```

Comment:
If `releaseColumn` is not `true` those latest releases won't be displayed.

Reply:
Forgot to commit that when I previewed.  Fixing!  Thanks for noticing! :-)